instrumentation/quarkus-resteasy-reactive/quarkus3-testing/build.gradle.kts

```diff
@@ -45,12 +45,20 @@ val quarkusTestBaseRuntimeClasspathConfiguration by configurations.creating {
 val quarkusTestCompileOnlyConfiguration by configurations.creating {
 }
 
-val testModelPath = layout.buildDirectory.file("quarkus-app-test-model.dat").get().asFile.toPath()
+val testModelPath = layout.buildDirectory.file("quarkus-app-test-model.dat")
 
-val buildModel = tasks.register("buildModel") {
-  dependsOn(configurations.named("testRuntimeClasspath"))
+val buildModel = if (findProperty("skipTests") as String? != "true") {
```

Comment:
is it possible to avoid this duplicated code with the change above?
maybe it could be in the conventions?